conan/tools/gnu/makedeps.py

```diff
@@ -67,6 +68,15 @@ def _makefy(name: str) -> str:
     return re.sub(r'[^0-9A-Z_]', '_', name.upper())
 
 
+def _makefy_properties(properties: dict) -> Optional[dict]:
```

Comment:
Maybe you can simplify the Optional by:
- Always return a dict, an empty one instead of None
- That also simplifies the template, that does no longer need to do ``{%- if values is not none -%}``

Reply:
done